### PR TITLE
[BUG] Crash when the token is expired

### PR DESCRIPTION
**Pull request by Aitorbp**
## Related Issues
App:

- [ ] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt [hunk 1]
```diff
@@ -537,12 +541,24 @@ class MainFileListFragment : Fragment(),
...
+                for (folderContent in fileListUiState.folderContent) {
+                    val newFile = folderContent.file.copy(previousUri = account?.let { mainFileListViewModel.getPreviewUriForFile(folderContent, it) })
+                    val newFolderContent = folderContent.copy(file = newFile)
+                    newFolderContentList.add(newFolderContent)
+                }
+                CoroutineScope(Dispatchers.Main).launch {
+                    fileListAdapter.updateFileList(
+                        filesToAdd = newFolderContentList,
+                        fileListOption = fileListUiState.fileListOption,
+                        imageLoader = mainFileListViewModel.getCoilImageLoader()
```
Comment (manuelplazaspalacio) [inline owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt:557]: A coma here.

File: owncloudDomain/src/main/java/com/owncloud/android/domain/files/model/OCFile.kt [hunk 2]
```diff
@@ -57,6 +57,7 @@ data class OCFile(
     var sharedWithSharee: Boolean? = false,
     var sharedByLink: Boolean = false,
     val spaceId: String? = null,
+    val previousUri: String? = ""
```
Comment (manuelplazaspalacio) [inline owncloudDomain/src/main/java/com/owncloud/android/domain/files/model/OCFile.kt:60]: Add a coma. Why the name is "previousUri"? I would put something like "thumbnailUri"

**Review by manuelplazaspalacio — CHANGES_REQUESTED** [review]
Some change here @Aitorbp

File: owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt (hunk 1 above)
Comment (Aitorbp) [reply owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt:557]: done

File: owncloudDomain/src/main/java/com/owncloud/android/domain/files/model/OCFile.kt (hunk 2 above)
Comment (Aitorbp) [reply owncloudDomain/src/main/java/com/owncloud/android/domain/files/model/OCFile.kt:60]: Done

**Review by manuelplazaspalacio — APPROVED** [review]

**Comment by jesmrec** [comment]
Rejected. Replaced by https://github.com/owncloud/android/pull/4132

**Pull request closed without merging**